apps/daffio/src/app/content/docs/api-docs/components/api-doc/api-doc.component.scss

```diff
@@ -0,0 +1,128 @@
+@import 'utilities';
+@import "theme";
+
+.api-doc {
+  margin: 25px;
+
+
+  & a {
+    color: daff-color($primary);
+  }
+}
+
+.api-header {
+  display: block;
+
+  @include breakpoint(mobile) {
+    display: flex;
+    align-items: center;
+  }
+
+  &__label {
+    @include embolden();
+    display: inline-block;
+    border-radius: 3px;
+    color: $white;
+    margin: 15px 0 0 0;
+    padding: 4px 8px;
+    text-transform: uppercase;
+    box-sizing: border-box;
+
+    @include breakpoint(mobile) {
+      margin: 0 0 0 15px;
+    }
+  }
+
+  &__label.class {
```

Comment:
All of this should just be `include @type-label();`

Reply:
done